packages/react/src/components/Storage/FileUploader/FileUploader.tsx

```diff
@@ -33,33 +34,59 @@ export function FileUploader({
     addTargetFiles,
     showPreviewer,
     setFiles,
-  } = useFileUploader();
+    fileStatuses,
+    setFileStatuses,
+    setFileSizeErrors,
+  } = useFileUploader(maxSize, acceptedFileTypes, multiple);
+
   const [allFileNames, setAllFileNames] = useState<string[]>([]);
-  const [fileStatuses, setFileStatuses] = useState<FileStatuses>([]);
   const fileStatusesRef = useRef<FileStatuses>([]);
   // File Previewer global states
   const [isLoading, setLoading] = useState(false);
   const [isSuccess, setSuccess] = useState(false);
   const [percentage, setPercentage] = useState(0);
+  // Tracker states
+  const [isEditingNames, setIsEditingNames] = React.useState<boolean[]>([]);
```

Comment:
`isEditingNames` as a name should be a singular boolean, suggest renaming to something that is more indicative of what it is actually being used for

Reply:
I'll rename to `isEditingName` singular